sessions/js-functions-1/vending-machine/README.md

```diff
@@ -0,0 +1,36 @@
+# JavaScript exercises for basic functions
+
+## Task
```

Comment:
students got confused in the past since there are not tasks. please refer here to the index. js